package.json

```diff
@@ -9,8 +9,10 @@
   },
   "author": "Mercury Squad",
   "scripts": {
-    "start": "nodemon index.js",
+    "start": "node index.js",
+    "dev": "nodemon index.js",
     "addData": "node scripts/importData.js",
+    "startInvoiceScheduler": "node src/scheduler/index.js",
```

Comment:
@AchillesKarki this script has to be executed as well additionally in the server ?

Reply:
oh yes I forgot to mention that. Also I have created another script for dev. `npm run start` should be executed in prod server, whereas `npm run dev` to be executed in local.